internal/impl/parquet/processor_decode.go

```diff
@@ -72,13 +87,27 @@ func init() {
 //------------------------------------------------------------------------------
 
 func newParquetDecodeProcessorFromConfig(conf *service.ParsedConfig, logger *service.Logger) (*parquetDecodeProcessor, error) {
+	handleLogicalTypes, err := conf.FieldString(pFieldHandleLogicalTypes)
+	if err != nil {
+		return nil, err
+	}
+
+	logicalTypesVersion, err := strconv.Atoi(handleLogicalTypes[1:])
```

Comment:
I know it's extra plumbing, but can we have a const for each version and use a `switch` statement? If not, users can pass arbitrary data here if running via the `--chilled` flag.

Reply:
Makes sense, turned it into a switch statement and made sure that invalid values throw an error.